applications/photo-asset-manager/cdk/lib/backend/strategies.ts

```diff
@@ -248,6 +249,26 @@ export const CPP_LAMBDAS_STRATEGY: PamLambdasStrategy = {
   },
 };
 
+export const PHP_LAMBDAS_STRATEGY: PamLambdasStrategy = {
+  ...EMPTY_LAMBDAS_STRATEGY,
+  memorySize: 256,
+  timeout: Duration.minutes(5),
+  codeAsset() {
+    let phpPAMZip = resolve(
+        "../../../php/applications/photo_asset_manager/bootstrap.zip"
```

Comment:
Where does this get created? Can we include that script in this PR?

Reply:
Good question. The creation of this zip file is a large undertaking which is out of scope _for now_. It might be revisited in the future if people find value in creating custom Lambda PHP runtimes.